### ISSUE TITLE
When training with the command "python rain.py env.num_envs=1968 env.num_obstacles=350 env_dyn.num_obstacles=80," the results are as shown below, which are not very good. How should I modify the parameters? Thanks

### ISSUE THREAD

**Issue by ZhangXiaochao94** [Zhefan-Xu/NavRL#145]
Hello everyone, I have a question to ask. Due to version issues, I removed wandb and switched to tensorboard for data recording. When training with the command "python rain.py env.num_envs=1968 env.num_obstacles=350 env_dyn.num_obstacles=80," the results are as shown below, which are not very good. How should I modify the parameters? Thank you
![Image](https://github.com/user-attachments/assets/5b5baedf-94fb-4ffa-8b3e-178bc1dfcb64)
![Image](https://github.com/user-attachments/assets/ed86c259-bbd8-438b-a68e-93f72c087ef2)
![Image](https://github.com/user-attachments/assets/c616aed0-f366-4b23-a6ec-195c012ec3d0)

**Comment by ZhangXiaochao94** [comment]
Due to input issues, the description just now was not accurate enough. The actual training command is: python train.py env.num_envs=1024 env.num_obstacles=350 env_dyn.num_obstacles=80  Or how can I reproduce the accuracy of 82.71 in the pape

**Comment by Dream-lixiang** [comment]
我是先按按照开源的超参数跑一轮，train/stats.reach_goal 是 72 左右 ，接着训练把学习率都改成 learning_rate: 2e-4 再训练一轮，train/stats.reach_goal能达到 81.9 ，差不多就能用了，注意只改他的学习率，别改entropy_loss_coefficient 这个，我试过把这个改小但是越训练越差

**Comment by ZhangXiaochao94** [comment]
> 我是先按按照开源的超参数跑一轮，train/stats.reach_goal 是 72 左右 ，接着训练把学习率都改成 learning_rate: 2e-4 再训练一轮，train/stats.reach_goal能达到 81.9 ，差不多就能用了，注意只改他的学习率，别改entropy_loss_coefficient 这个，我试过把这个改小但是越训练越差

Thanks！

**Comment by li1724074356-ship-it** [comment]
> 我是先按按照开源的超参数跑一轮，train/stats.reach_goal 是 72 左右 ，接着训练把学习率都改成 learning_rate: 2e-4 再训练一轮，train/stats.reach_goal能达到 81.9 ，差不多就能用了，注意只改他的学习率，别改entropy_loss_coefficient 这个，我试过把这个改小但是越训练越差

大佬你好这个学习率实在ppo.yaml里改的吗

**Comment by Dream-lixiang** [comment]
对的，改这个ppo.yaml和train.yaml配置文件，然后要接着训练的话注意把train.py里面注释掉的权重地址加上换成自己的，剩下就是步数加倍，不然接着训一看步数到了就自己停止了，总之改完让ai再看看基本没什么问题

**Comment by li1724074356-ship-it** [comment]
> 的，改这个ppo.yaml和train.yaml配置文件，然后要继续的话训练注意把train.py里面注释掉的权重地址加上换成自己的，剩下就是步数加倍，不然接着训阿步数到了就自己停止了，总之改完让ai再看看基本没什么问题

好的谢谢指导
